src/main/java/com/hyperwallet/clientsdk/model/HyperwalletHateoasLink.java

```diff
@@ -0,0 +1,30 @@
+package com.hyperwallet.clientsdk.model;
+
+import java.util.List;
+import java.util.Map;
+
+public class HyperwalletHateoasLink{
```

Comment:
can we name this as `HyperwalletLink` and in Javadoc just declare that this represents `HATEOAS`

Reply:
Updated the class name.